frontend/src/components/SchedulerCalendar_legacy.js

```diff
@@ -0,0 +1,223 @@
+import React, { Component } from "react";
```

Comment:
Czy potrzebujemy ten plik?
Jeśli dobrze rozumiem, to jest on kopią poprzedniego SchedulerCalendar...

Reply:
Usuniete